packages/node_modules/@webex/plugin-rooms/src/rooms.js

```diff
@@ -37,8 +37,28 @@ const debug = require('debug')('memberships');
  */
 const Rooms = WebexPlugin.extend({
   /**
-   * Connect to the web socket to listen to incoming messages.
+   * Register to listen for incoming rooms events
+   * This is an alternate approach to registering for rooms webhooks.
+   * The events passed to any registered handlers will be similar to the webhook JSON,
+   * but will omit webhook specific fields such as name, secret, url, etc.
+   * See the <a href="https://js.samples.s4d.io/browser-socket/">Sample App</a>
+   * for more details.
+   * @instance
+   * @memberof Rooms
    * @returns {Promise}
+   * @example
+   * webex.rooms.listen()
+   *   .then(() => {
+   *     console.log('listening to room events');
```

Comment:
Use logger instead.

Reply:
Those console.log statements are in the JSDoc example. They aren't actually called they are there to show a developer how to use the listen() method in their app (which may not have a logger).